### PR TITLE
Maintenance page init

### PR DESCRIPTION
**Pull request by Francois18**
The maintenance page is initialized and can be accessed by routing to  **/maintenance**

Screenshot (2)
Screenshot (3)

### CODE REVIEW

File: components/Maintenance/Maintenance.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+} from '../modules/__modules/Vectors/index'
+
+const Maintenance = () => {
+    const router = useRouter();
+
+    return (
+        <div className='flex flex-wrap min-h-screen items-center justify-center '>
+            <div className=' max-w-6xl p-4'>
+                <div className=' flex flex-col items-center justify-center text-center ' >
+                    <img src="https://templates.iqonic.design/muzik/html/images/error/02.png" alt="logo" className='sm:h-56 h-28  mb-6' />
```
Comment (Jonath-z) [inline components/Maintenance/Maintenance.jsx:16]: image or svg here , dowload it then use it

File: components/Maintenance/Maintenance.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                    <h1 className='font-extrabold text-4xl p-4'>We are Currently Performing Maintenance</h1>
+                    <p className='font-lighter mb-4 p-2'>Please chack back sometime</p>
+                </div>
+                <div className='grid lg:grid-cols-3 gap-6 justify-center place-items-center text-center md:px-20'>
+
+                    <div className=' rounded-lg p-4 m-1 md:w-full shadow-xl lg:shadow-2xl'>
+                        <div className='px-4 py-2 flex flex-col justify-center items-center'>
+                            <span className='text-primary '>
+                                < LayoutTopLine size={54} />
```
Comment (Jonath-z) [inline components/Maintenance/Maintenance.jsx:25]: remove empty space

File: components/Maintenance/Maintenance.jsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+                        <div className='px-4 py-2 flex flex-col justify-center items-center'>
+                            <span className='text-primary '>
+                                <Clock size={54} />
+                            </span>
+                            <h2 className='font-bold text-lg p-2'>What is the Downtime?</h2>
+                            <p className='text-gray-600 font-light'>It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.</p>
+                        </div>
+                    </div>
+                    <div className=' rounded-lg p-4 m-1 md:w-full shadow-xl lg:shadow-2xl'>
+                        <div className='px-4 py-2 flex flex-col justify-center  items-center '>
```
Comment (Jonath-z) [inline components/Maintenance/Maintenance.jsx:41]: create a card for this maintance motifs as they have the same design.

**Review by Jonath-z — CHANGES_REQUESTED** [review]
kindly applie these changes

when you already create the folder like **Maintenace** don't name the main file inside `Maintenace` again. just name it `index.jsx`. this can be applied in `components/Maintenance`

**Review by Jonath-z — APPROVED** [review]
LGTM

**Comment by Jonath-z** [comment]
try to resolve conflict

**Comment by Francois18** [comment]
> try to resolve conflict

resolved

**Pull request merged**